cms/server/ContestWebServer.py

```diff
@@ -135,10 +136,15 @@ def prepare(self):
         self.r_params = self.render_params()
 
     def get_current_user(self):
-        """Gets the current user logged in from the cookies
+        """Gets the current user and participation from cookies.
 
-        If a valid cookie is retrieved, return a User object with the
-        username specified in the cookie. Otherwise, return None.
+        If a valid cookie is retrieved, return a (User, Participation)
```

Comment:
Isn't it easier to just return the participation and let the caller use participation.user, which is already loaded since we are in the same session?